src/components/templates/Jazzbar/JazzBar/JazzBar.module.scss

```diff
@@ -0,0 +1,14 @@
+@use "scss/attendee/font";
+@use "scss/attendee/space";
+
+.componentSpaceInfo {
+  text-align: center;
+  padding: 0 space.empty(1);
+  margin: 0 auto;
+  width: 100%;
+  max-width: 420px;
```

Comment:
[NOTE]
Awfully close to `layout.$w-sidebar`. If that's not the intent, it sounds like it's `space.item(10)`.

Reply:
Created a new variable in `layout`. See 22c9bf0fd

Reply:
The idea I had with that file was: 

layout is for "big things" that we need to sync across the code-base, a few well defined parameters in the style like like (I'm guessing here) a width and height of a space component, a card, a chat width and similar.

All else might as well go into `$-private-constants`